qna/AT-Sorglos-Vorteile-für-KassenBetreiber.md

```diff
@@ -20,118 +20,123 @@ lang-de, market-at, portal, webshop, bundle-carefree, consulting, documentation,
+| - Die fiskaltrust.Middleware (Software) muss im Firmennetzwerk installiert werden und benötigt Zugriff auf den USB-Stick. <br />- Die Sicherheitseinrichtung (z.B. SmartCard+USB Stick) muss installiert werden (bei virtuellen Servern oft nicht so einfach)<br />- Die Sicherheitseinrichtung (SmartCard) muss personalisiert werden.<br />- Die fiskaltrust.Middleware muss (z.B. mit einem Template) für jede Kasse erstellt werden.<br />- Dies ermöglicht meist stabile und gleichmäßige Signatur-Zeiten | - Keine lokale Installation notwendig<br />- eine stabile Internetverbindung ist vorteilhaft um Ausfälle möglichst zu vermeiden<br />- Einfach im Shop zu erwerben und einzurichten<br />- Die Signatur-Zeiten sind von der Internetverbindung abhängig<br />- Hohe Performance (Signaturen pro Sekunde) möglich |
 
 #### **VIELE HERSTELLER HINTERLEGEN GENAU DIE RICHTIGE PRODUKT-VARIANTE FÜR IHRE SOFTWARE BEI UNS IM SHOP.**
 
-#### **WENN IHR HERSTELLER EIN PRODUKT EMPFIEHLT, SO SOLLTEN SIE GENAU DIESES PRODUKT ERWERBEN.**
+#### **WENN IHR HERSTELLER EIN PRODUKT EMPFIEHLT, SO SOLLTEN SIE GENAU DIESES PRODUKT ERWERBEN UM DIE INSTALLATION ZU VEREINFACHEN.**
 
-Wenn Sie Ihre Kassa gleich bei Fiskaltrust gesetzeskonform bei Finanz Online anmelden möchte, so benötigen Sie folgende Produkte:
+Zur gesetzeskonformen Anmeldung Ihrer Kassa durch fiskaltrust bei FinanzOnline benötigen Sie folgende Produkte:
 
-- 1x FinanzOnline-Meldung SEE #15 Produkt #215
-- 1x FinanzOnline-Meldung RK #16 Produkt #216
-- 1x FinanzOnline-Belegprüfung #17 FinanzOnline-Belegprüfung #217
+- 1x [FinanzOnline-Meldung SEE #215](https://www.fiskaltrust.at/products/product_01/#1468154597168-53fabb3d-7842)
+- 1x [FinanzOnline-Meldung RK #216](https://www.fiskaltrust.at/products/product_01/#1468159113106-9607b186-deeb)
+- 1x [FinanzOnline-Belegprüfung #217](https://www.fiskaltrust.at/products/product_01/#1468159257275-22b0c802-9330)
 
-Alternativ werden für Besitzer des **Sorglos.Paketes** Finanz Online Meldungen unlimitiert durchgeführt*
+Alternativ werden für Besitzer eines **Sorglos.Paket** ([#4154-0201](https://www.fiskaltrust.at/products/product_01/#1468153721989-91b86112-b158), [#4154-0207](https://www.fiskaltrust.at/products/product_01/#1475349131348-1769c5fd-b93f)) FinanzOnline-Meldungen ([#215](https://www.fiskaltrust.at/products/product_01/#1468154597168-53fabb3d-7842), [#216](https://www.fiskaltrust.at/products/product_01/#1468159113106-9607b186-deeb), [#217](https://www.fiskaltrust.at/products/product_01/#1468159257275-22b0c802-9330)) unlimitiert durchgeführt*
 
 #### **Wer hilft mir bei der Entscheidung?**
 
-##### Mein Kassenhersteller, bzw. mein Kassenhändler
+##### Mein KassenHersteller, bzw. mein KassenHändler
```

Comment:
Schreiben wir das jetzt in Österreich auch so? Ich dachte wir bleiben hier bei der klassischen Schreibweise bzw. den englischen Begriffen?